### ISSUE TITLE
No overview of the installation process

### ISSUE THREAD

**Issue by clissold345** [canonical/ubuntu.com#1283]
## Summary

www.ubuntu.com doesn't seem to give an overview of the installation process. I was looking at desktop installation.

## Process

[list the steps to replicate the issue]

## Current and expected result

I can see pages for all the elements of installation:

https://www.ubuntu.com/download/desktop
https://www.ubuntu.com/download/desktop/burn-a-dvd-on-ubuntu
https://www.ubuntu.com/download/desktop/create-a-usb-stick-on-ubuntu
https://www.ubuntu.com/download/desktop/try-ubuntu-before-you-install
https://www.ubuntu.com/download/desktop/install-ubuntu-desktop

but there doesn't seem to be a page - and I think it should a prominent page - that overviews the whole process.

I believe the process is typically as follows:

- Check the recommended system requirements
- Download the ISO.
- Burn a DVD or create a bootable USB stick
- Boot from the DVD or USB stick to check your computer is compatible with Ubuntu
- Back up your important files (in the unlikely event that the installation fails)
- Install Ubuntu.

I think it would be useful (certainly for people who haven't done an installation before) to have a prominent overview page on www.ubuntu.com.

## Screenshot

[if relevant, include a screenshot]

**Comment by pmahnke** [comment]
@clissold345 

That is certainly the intention of the https://www.ubuntu.com/download/desktop/install-ubuntu-desktop page.

It doesn't have some of the steps you mention as they are on other pages... like the requirements on the /download/desktop pages

![image](https://cloud.githubusercontent.com/assets/441217/22375944/fbfd0a36-e4a3-11e6-9523-08b5525ae94d.png)


and the info on dvds or usb sticks on /download/desktop

![image](https://cloud.githubusercontent.com/assets/441217/22375962/16e97172-e4a4-11e6-8fb0-185a36545419.png)

If you want to make some suggestions to fix up the install-ubuntu-desktop page, or even a PR, that would be great.  I think a live boot check and a reminder to back-up are worth adding.

Thanks,

Peter

**Comment by clissold345** [comment]
Hi Peter

Yes the install-ubuntu-desktop page is quite a good "overview" page. As you say, it doesn't cover all the points I listed. install-ubuntu-desktop doesn't seem very easy to find on www.ubuntu.com. Can it be made easier to find? In my opinion it's a very important page for anyone who wants to install Ubuntu desktop.

When you say "If you want to make some suggestions to fix up the install-ubuntu-desktop page ..." do you mean add another comment to this issue? Or something else? When you say "PR" (=problem report?) do you mean create an ubuntu bug? I know how to do that but it always asks for a package and I don't know what package to assign in this case. If a PR isn't an ubuntu bug then I don't know what you mean. Yes I'm willing to help but I need clarification on what to do.

**Comment by nottrobin** [comment]
Hi @clissold345.

PR means a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) againt this very repository. It would allow you to show us exactly how you would restructure the page or the site to solve this issue.

You can create one by editing [this page](https://github.com/ubuntudesign/www.ubuntu.com/blob/master/templates/download/desktop/install-ubuntu-desktop.html) or any other page in the repisitory, while signed in to GitHub.

If you don't have time or wish to do that, it would also be helpful if you could simply lay out exactly how you would reword or restructure the pages on the site as a new comment on this very issue.

**Comment by clissold345** [comment]
Hi notrobin, thanks for your reply. Regards, Chris.

Hi guys, I've written a draft in English. Would anyone be willing to edit install-ubuntu-desktop for me and submit a Pull Request? Regards, Chris.

[ Draft page ]

Install Ubuntu Desktop 16.04 LTS

Step 1 Check system requirements.

Recommended system requirements:

    2 GHz dual core processor or better
    2 GB system memory
    25 GB of free hard drive space
    Either a DVD drive or a USB port for the installer media
    Internet access is helpful

If your computer matches or exceeds the above requirements it should be fine for moderately heavy day-to-day use of Ubuntu.

Step 2 Download the ISO image.

[link to https://www.ubuntu.com/download/desktop

This isnt a great link because it repeats the system requirements, which the user has already seen - but it will do. ]

Step 3 Burn a DVD or create a bootable USB stick.

Burn a DVD:

[ link to https://www.ubuntu.com/download/desktop/burn-a-dvd-on-ubuntu ]

or create a bootable USB stick:

[ link to https://www.ubuntu.com/download/desktop/create-a-usb-stick-on-ubuntu ]

Step 5 Run Ubuntu from the DVD or USB stick.

This allows Ubuntu to perform checks to see if your computer is compatible with Ubuntu.

[ Insert link to https://www.ubuntu.com/download/desktop/try-ubuntu-before-you-install ]

Step 6 Back up your important files.

If you have any important files on your computer, it's recommended that you back them up. One way of doing this is to copy the files to a USB stick.
The reason for backing them up is that, in the unlikely event that the install fails, the files on your computer may be lost.

[ Please delete the old step 1. It's not needed any more. Then the text of the rest of the page is the same as before - except
  of course that the steps are now renumbered 7 "Prepare to install Ubuntu" to 14 "That's it", 
  and of course the tip at the bottom of the page stays in too. ]

**Comment by deadlight** [comment]
We will see if we can use some or all of this content on the page.
Thanks,
Karl

**Comment by anasereijo** [comment]
@pmahnke this issue is 4 years old. Do you think it's still relevant or can we close it?

**Comment by sowasred2012** [comment]
@pmahnke do you think this is still valid?
